### PR TITLE
Static reflection

### PR DESCRIPTION
**Pull request by MartinMystikJonas**

### CODE REVIEW

File: src/Template/Variable.php [hunk 1]
```diff
@@ -24,6 +24,11 @@ public function getName(): string
         return $this->name;
     }
 
+    public function getType(): Type
```
Comment (lulco) [inline src/Template/Variable.php:27]: do we need this? it is not used

**Comment by lulco** [comment]
please fix sniffer

File: src/Template/Variable.php (hunk 1 above)
Comment (MartinMystikJonas) [reply src/Template/Variable.php:27]: shit it was commited by mistake. I need it in that variable type merging I was working on and I commited it in this PR by mistake. Should I remove it or will we keep it here?

**Comment by MartinMystikJonas** [comment]
@lulco CS fixed

File: src/Template/Variable.php (hunk 1 above)
Comment (lulco) [reply src/Template/Variable.php:27]: If you will need it later, we can merge

**Pull request merged**